dev-packages/browser-integration-tests/suites/tracing/linking-addLinks/test.ts

```diff
@@ -0,0 +1,80 @@
+import { expect } from '@playwright/test';
+import type { Event, SpanJSON } from '@sentry/core';
+import { sentryTest } from '../../../utils/fixtures';
+import { getMultipleSentryEnvelopeRequests, shouldSkipTracingTest } from '../../../utils/helpers';
+
+sentryTest('should link spans with addLinks() in trace context', async ({ getLocalTestUrl, page }) => {
+  if (shouldSkipTracingTest()) {
+    sentryTest.skip();
+  }
+
+  const url = await getLocalTestUrl({ testDir: __dirname });
+  const [rootSpan1, rootSpan2, rootSpan3] = await getMultipleSentryEnvelopeRequests<Event>(page, 3, { url });
```

Comment:
If I understand this test correctly, we're sending 4 transaction events here, right? waiting on 3 then also introduces a potential flake as we depend on events being sent in the correct order.